app/src/test/java/gov/va/vro/routes/MasIntegrationRoutesTest.java

```diff
@@ -96,7 +96,7 @@ private MasProcessingObject processClaim(Boolean sufficientEvidence) throws Exce
 
     // Mock a return value when claim-submit (lighthouse) is invoked
```

Comment:
i'd..fix the comment here.

Reply:
Okay np, I'll make it claim-submit-full instead.